packages/react-composites/src/composites/common/AddPeopleDropdown.tsx

```diff
@@ -60,6 +63,12 @@ export const AddPeopleDropdown = (props: AddPeopleDropdownProps): JSX.Element =>
       useTargetWidth: true,
       calloutProps: {
         preventDismissOnEvent
+      },
+      onMenuOpened(contextualMenu?) {
+        console.log('actioned' + announcerStrings);
+        if (announcerStrings !== undefined) {
+          setAnnouncerStrings(undefined);
```

Comment:
nit: simplify if statement to
`setAnnouncerStrings(announcerStrings ?? undefined)`

Reply:
For reference:
https://dev.to/sunflower/reactjs-if-it-is-setting-a-state-with-the-same-value-will-the-component-be-re-rendered-5g24 
undefined is needed here as the function component would not be re-rendered if it was set to the same value down in line 82. Meaning, the announcer would not announce again if the copy invite link was actioned on more than once unless the announcerString is reset like in this case.

Reply:
from testing this does work this way, but I will change it since yours is more readable